lib/galaxy_test/api/test_workflows.py

```diff
@@ -967,6 +969,15 @@ def test_url_import(self):
             workflow.get("source_metadata").get("url") == url
         )  # disappearance of source_metadata on modification is tested in test_trs_import
 
+    def test_base64_import(self):
+        base64_url = "base64://" + base64.b64encode(workflow_str.encode("utf-8")).decode("utf-8")
+        response = self._post("workflows", data={"archive_source": base64_url})
+        print(response.content)
```

Comment:
Is this print intended or a debug leftover?

Reply:
Not intended - a debug leftover. I'll remove it in a follow up.

Reply:
Cleaned up in https://github.com/galaxyproject/galaxy/pull/13949/commits/0800be183cd49e760618718e61a4b52eb187872a now as part of #13949. Thanks for merging!